src/main/webapp/resources/js/components/alerts/ErrorAlert.jsx

```diff
@@ -1,19 +1,35 @@
+/**
+ * Component to render an Error alert with an icon
+ */
+
 import React from "react";
 import PropTypes from "prop-types";
 import { Alert } from "antd";
 
-export function ErrorAlert(props) {
+/**
+ * Stateless UI component for displaying an [antd error Alert]{@link https://ant.design/components/alert/}
+ *
+ * @param {string} message - Text to display in alert
+ * @param {string} description - Optional description
+ *
+ * @returns {antd Alert} - Returns an antd error 'Alert' component
```

Comment:
This is not a proper return type.  Take a look at JSDoc, but I think you can use `element`